app/services/green_lanes/determine_assessments.rb

```diff
@@ -0,0 +1,41 @@
+module GreenLanes
+  class DetermineAssessments
```

Comment:
This doesn't have the api of a service object and it feels more like a presenter. Given its just a presenter, I'm wondering if we can avoid the side effects of calling determine_assessments in the constructor and just pull the answers to these specific questions on read?

Reply:
```ruby
class CategoryAssessmentsPresenter
  def initialize(goods_nomenclature, ans)
    @goods_nomenclature = goods_nomenclature
    @ans = ans
  end

  def category_1_assessments
    assessments.category_1_assessments
  end 

  def assessments
    @assessments ||= GreenLanes::DetermineAssessments.new(@goods_nomenclature, @ans)
  end
end
```

Reply:
Or better yet, just present the result of the assessments

```ruby
class CategoryAssessmentsPresenter
  def initialize(assessments)
    @assessments = assessments
  end

  def category_1_assessments
    assessments.category_1_assessments
  end

  private

  attr_reader :assessments
end
```